docs/source/crud_events.rst

```diff
@@ -0,0 +1,180 @@
+CRUD Events
+===========
+
+Ramses supports Nefertari CRUD events. Following documentation describes how to define and connect them.
+
+
+Writing Processors
+------------------
+
+You can write custom functions inside your ``__init__.py`` file, then add the ``@registry.add`` decorator before the functions that you'd like to turn into CRUD event subscriber.
+CRUD event subscribers must have the same API as Nefertari CRUD event subscribers. Check Nefertari CRUD Events doc on more details on events API.
+
+E.g.
+
+.. code-block:: python
+
+    @registry.add
+    def lowercase(event):
+        """ This processor lowercases the value of a field """
+        value = (event.field.new_value or '').lower().strip()
+        event.set_field_value(value)
+
+
+Connecting event subscribers
+----------------------------
+
+When you defined event subscribers in your ``__init__.py`` as described above, you can connect them per-model or per-modelfield.
+
+In general subscribers are set up using ``_event_handlers`` param. Value of this param is an object, keys of which are so called "event tags" and values are list of subscriber names.
+Event tags are constructed of two parts: ``<type>_<action>`` where:
+
+**type**
+    Is either ``before`` or ``after``, depending on when subscriber should run - before view method call or after respectively.
+**action**
+    Exact name of Nefertari view method that processes the request (action)
+
+Complete list of actions:
+    * **index** - Collection GET
+    * **create** - Collection POST
+    * **update_many** - Collection PATCH/PUT
+    * **delete_many** - Collection DELETE
+    * **collection_options** - Collection OPTIONS
+    * **show** - Item GET
+    * **update** - Item PATCH
+    * **replace** - Item PUT
+    * **delete** - Item DELETE
+    * **item_options** - Item OPTIONS
+
+And one special action:
+    * **set** - includes actions ``create``, ``update``, ``replace``, ``update_many``
+
+
+E.g. following connects ``lower_strip_processor`` subscriber to ``after_index`` event and the subscriber will be run after collection GET request is processed.
```

Comment:
this is a bad example because it doesn't make sense. you want to use the `before_update` or `before_create` or better yet `set` event for this handler.